python/tests/conftest.py

```diff
@@ -166,7 +164,11 @@ def _version_store_factory_impl(
     name = name or default_name
     if name == "_unique_":
         name = name + str(len(used))
-    assert (name not in used) or reuse_name, f"{name} is already in use"
+    if name in used:
+        if reuse_name:
+            return used[name]  # Windows does not allow LMDB open mmap with the same path twice
```

Comment:
The point of using `reuse_name` is to have a different library config while reusing the underlying data.

This change is not doing the former.

Reply:
Sorry it was trying to get away from the lmdb open mmap file issue in Windows. It is a newly introduced limitation, which makes a test fixture unable to be passed more than one lmdb storage fixture.
I guess I have to bite the bullet and re-design the defragmentation test case.